src/rpc/mining.cpp

```diff
@@ -126,17 +126,21 @@ static UniValue generatetoaddress(const JSONRPCRequest& request)
 
 UniValue getnewblock(const JSONRPCRequest& request)
 {
-    if (request.fHelp || request.params.size() > 2)
+    if (request.fHelp || request.params.size() > 3)
         throw std::runtime_error(
                 "getnewblock\n"
                 "\nGets hex representation of a proposed, unmined new block\n"
                 "\nArguments:\n"
-                "1. address         (string, required) The address to send fees and the newly generated coin.\n"
-                "2. required_age    (numeric, optional, default=0) How many seconds a transaction must have been in the mempool to be inluded in the block proposal. This may help with faster block convergence among functionaries using compact blocks.\n"
+                "1. address                    (string, required) The address to send fees and the newly generated coin.\n"
+                "2. required_age               (numeric, optional, default=0) How many seconds a transaction must have been in the mempool to be inluded in the block proposal. This may help with faster block convergence among functionaries using compact blocks.\n"
+                "3. xfield_type:new_xfield_value (multiple, optional) Change network wide value of the xfield parameter from the next block. The xfield parameter is identified by the xfield_type.\n"
+                " xfield_type supported are:\n"
+                "    1 : Aggregate Public Key of the Tapyrus Signer network.  (string )\n"
                 "\nResult\n"
                 "blockhex      (hex) The block hex\n"
                 "\nExamples:\n"
                 + HelpExampleCli("getnewblock", "")
```

Comment:
In this case, cli output following error. so it should be non-error example.

```./src/tapyrus-cli getnewblock ""                                   
error code: -5
error message:
Error: Invalid address
```

Reply:
The problem is PROD and DEV modes. The address was valid in PROD mode:

>  % ./src/tapyrus-cli --conf=/Users/navia/tapyrus.conf -datadir=/Users/navia/.tapyrus getnewblock 1DV3jX8bujEW4vAUYgsKDNa8UwAxAXGiEb
>01000000e679f091aec5bcf3b8b97e2b5c56e654ab029d35d54ac4cff50ee3d980b72e66bfdf3c047031ecfc61c10600cbb368713af9e4c03a959a90ea60a01e74584344e2094edbf2ec728515a9f1636b69dcd35c2273cebd07d2e83edf44e77cb6748864873663000001010000000100000000000000000000000000000000000000000000000000000000000000000100000003510101ffffffff0100f2052a010000001976a91488ef70143151ed15b805bda520fc16a43326fbb288ac00000000

Reply:
I see.